### ISSUE TITLE
Not following all rules

### ISSUE THREAD

**Issue by KrekkieD** [SublimeLinter/SublimeLinter-jshint#36]
It appears as though the whitespace related rules ('indent', 'white') are not showing errors in the gutter. This is since I upgraded the package today, it used to work properly. It does trigger errors when I change the quotmark property, so I'm sure the .jshintrc file is loaded. Running sublime-text 3056 and the latest build through package control.

Moving the properties to the JSHint Gutter .jshintrc does not work either.

If I run jshint through grunt it does find the errors.

My .jshintrc file:

```
{
    "browser": true,
    "camelcase": true,
    "eqeqeq": true,
    "curly": true,
    "forin": true,
    "immed": true,
    "latedef": true,
    "newcap": true,
    "noarg": true,
    "indent": 4,
    "quotmark": "single",
    "undef": true,
    "unused": true,
    "trailing": true,
    "strict": true,
    "maxlen": 300,
    "white": true,
    "globals": {
        "define": true,
        "require": true
    }
}
```
